pkg/grafana/dashboard-handler.go

```diff
@@ -63,9 +66,26 @@ func (h *DashboardHandler) newDashboardResource(path, uid, filename string, boar
 	return resource
 }
 
+func (h *DashboardHandler) newDashboardFolderResource(path, folderName string) grizzly.Resource {
+	resource := grizzly.Resource{
+		UID:      folderName,
+		Filename: folderName,
+		Handler:  h,
+		Detail:   "",
+		Path:     path,
+	}
+	return resource
+}
+
 // Parse parses an interface{} object into a struct for this resource type
 func (h *DashboardHandler) Parse(path string, i interface{}) (grizzly.ResourceList, error) {
 	resources := grizzly.ResourceList{}
+	if path == dashboardFolderPath {
+		folderName := strings.ReplaceAll(i.(string), "{ }", "") // No idea why json parsing adds { } to the end of the parsed string :-(
```

Comment:
Might want to have a look at https://github.com/grafana/tanka/pull/358, looks like a similar problem.